apps/src/templates/maker/DiscountCodeSchoolChoice.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component, PropTypes } from 'react';
+import i18n from "@cdo/locale";
+import Button from "../Button";
+import SchoolAutocompleteDropdownWithLabel from '../census2017/SchoolAutocompleteDropdownWithLabel';
+import { styles as censusFormStyles } from '../census2017/censusFormStyles';
+
+const styles = {
+  confirmed: {
+    marginBottom: 5
+  },
+  button: {
+    marginTop: 10
+  }
+};
+
+export default class DiscountCodeSchoolChoice extends Component {
+  static propTypes = {
+    initialSchoolId: PropTypes.string,
+    initialSchoolName: PropTypes.string,
+    schoolConfirmed: PropTypes.bool.isRequired,
+    onSchoolConfirmed: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      confirming: false,
+      confirmed: props.schoolConfirmed,
+      schoolId: props.initialSchoolId,
+      schoolName: props.initialSchoolName,
+    };
+  }
+
+  handleDropdownChange = (field, event) => {
+    if (field === 'nces') {
+      this.setState({
+        schoolId: event.value,
+        schoolName: event.label,
+      });
+    }
+  }
+
+  handleClickConfirmSchool = () => {
+    this.setState({
+      confirming: true
+    });
+
+    // TODO: eventually this will be an API call. To start with, just fake it
+    // by at least making it async
+    setTimeout(() => {
+      this.setState({
+        confirming: false,
+        confirmed: true,
+      });
+      const fullDiscount = false;
+      this.props.onSchoolConfirmed(fullDiscount);
```

Comment:
Later we will POST to the server, and it will tell us whether we get a full discount or a partial discount.